### PR TITLE
Trunk 5371:Add Method level comments for missing methods in Format class

### PR DESCRIPTION
**Pull request by NYESIGA**
## Issue I worked on

see https://issues.openmrs.org/browse/TRUNK-

## Checklist: I completed these to help reviewers :)

- [ ] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [ ] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by NYESIGA** [comment]
see the https://issues.openmrs.org/browse/TRUNK-5371

**Comment by NYESIGA** [comment]
please someone to review this

**Comment by jwnasambu** [comment]
Kindly fix the ticket Id and also give a brief description of the changes you made please!

**Comment by NYESIGA** [comment]
The ticket Id TRUNK-5371 i added doc comments on formatPercentage() method in Format.java class

File: api/src/main/java/org/openmrs/util/Format.java [hunk 1]
```diff
@@ -28,11 +28,18 @@ public enum FORMAT_TYPE {
...
 	public static String formatPercentage(double pct) {
 		return NumberFormat.getPercentInstance().format(pct);
 	}
-	
+
+
+	/**
+	 * this method formats numbers to Percentage
+	 * @param  pct
+	 * @return it is static util method that allows to return null value without exception
```
Comment (tendomart) [inline api/src/main/java/org/openmrs/util/Format.java:41]: make the grammar proper

File: api/src/main/java/org/openmrs/util/Format.java [hunk 2]
```diff
@@ -28,11 +28,18 @@ public enum FORMAT_TYPE {
...
+
 	
 	public static String formatPercentage(double pct) {
 		return NumberFormat.getPercentInstance().format(pct);
 	}
-	
+
+
+	/**
+	 * this method formats numbers to Percentage
```
Comment (tendomart) [inline api/src/main/java/org/openmrs/util/Format.java:39]: This  formats numbers to Percentages

File: api/src/main/java/org/openmrs/util/Format.java (hunk 1 above)
Comment (NYESIGA) [reply api/src/main/java/org/openmrs/util/Format.java:41]: Ok thanks @tendomart   let me improve it

**Comment by dkayiwa** [comment]
@NYESIGA are you still working on this?

**Pull request closed without merging**